lib/action_account.rb

```diff
@@ -0,0 +1,148 @@
+require_relative './script_base'
+
+class ActionAccount < ScriptBase
+  # @api private
+  # A subtask is a class that has a run method, the type signature should look like:
+  # +#run(args: Array<String>, config: Config) -> Result+
+  # @return [Class,nil]
+  def subtask(name)
+    {
+      'review-reject' => ReviewReject,
+      'review-pass' => ReviewPass,
+    }[name]
+  end
+
+  def option_parser
```

Comment:
pretty sure this can be moved to the shared class, maybe we pass in the example banner as an arg